x/leverage/keeper/oracle.go

```diff
@@ -23,7 +23,7 @@ func (k Keeper) TokenPrice(ctx sdk.Context, denom string) (sdk.Dec, error) {
 
 	price, err := k.oracleKeeper.GetExchangeRateBase(ctx, denom)
 	if err != nil {
-		return sdk.ZeroDec(), err
+		return sdk.ZeroDec(), sdkerrors.Wrap(err, "oracle")
```

Comment:
I find myself having to use the rule "denom in error message is capitalized, so the error must be coming from oracle" a lot.

It will be better to prefix this one error return with a label since it's the only place where leverage calls an oracle function.